File: Iniciante/1008.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    int n, h;
    double v;
    cin >> n >> h >> v;
    std::cout << std::fixed << std::setprecision(2);
    cout << "NUMBER = " << n << endl; 
    cout << "SALARY = U$ " << (h * v) << endl;
    return 0;
}

File: Iniciante/C#/1008.cs
using System;
class URI {
    static void Main(string[] args) { 
        int n , h;
        double v;
        n = int.Parse(Console.ReadLine());
        h = int.Parse(Console.ReadLine());
        v = double.Parse(Console.ReadLine());
        Console.WriteLine("NUMBER = " + (n));
        Console.WriteLine("SALARY = U$ " + $"{(h*v):0.00}");
    }
}